cupyx/scipy/sparse/_index.py

```diff
@@ -14,7 +14,7 @@
 import numpy
 
 try:
-    import scipy
+    import scipy.sparse
```

Comment:
This change is because `scipy.sparse` is not imported when top-level `import scipy` is done. I think it happens to work as it was because `base.py` does call `import scipy.sparse`.